CTP/shell/src/com/navercorp/cubridqa/shell/main/Context.java

```diff
@@ -236,7 +235,41 @@ public String getTestCaseWorkspace() {
 			return ws;
 		}
 	}
-
+	
+	public String getTestPlatform(){
+		return getProperty("main.testing.platform", "linux");
+	}
+	
+	public String getTestCategory()
+	{
+		return getProperty("main.testing.category", "general");
```

Comment:
general -> shell.

Because main.testing.category has been removed from shell.conf, 'shell' is required.